agent-tee-phala/src/services/teeClient.ts

```diff
@@ -0,0 +1,32 @@
+import { TappdClient } from "@phala/dstack-sdk";
+import { privateKeyToAccount } from "viem/accounts";
+import { keccak256 } from "viem";
+
+export class TeeClient {
+    private client: TappdClient;
+
+    constructor() {
+        this.client = new TappdClient(process.env.DSTACK_ENDPOINT);
+    }
+
+    async generateAgentKeys(): Promise<{ agentPublicKey: string }> {
+        // Generate a unique path for key derivation
+        const uniquePath = `/keys/${Date.now()}-${Math.random().toString(36).substring(2)}`;
```

Comment:
this generates the same key given a {instanceId, path} set -- i need to esure the path contains some secret only the agent owner knows (which would be achieve through encryption based on the PK from the phala instance)